librosa/core/notation.py

```diff
@@ -671,22 +676,22 @@ def key_to_notes(key: str, *, unicode: bool = True) -> List[str]:
     `A♯:min` will use sharps, but spell note 0 (`C`) as `B♯`
 
     >>> librosa.key_to_notes('A#:min')
-    ['B♯', 'C♯', 'D', 'D♯', 'E', 'E♯', 'F♯', 'G', 'G♯', 'A', 'A♯', 'B']
```

Comment:
These docstring examples should be reverted if `natural=False` is default, and we should add a case or two to cover the `natural=True` setting.